File: codewars/Build Tower/BuildTower.js
function towerBuilder(nFloors) {
    let listy = [];
    let mxWid = nFloors * 2 - 1;
    for (let i = 0; i < nFloors; i++) {
        let stars = i * 2 + 1;
        let spaces = (mxWid - stars) / 2;
        let level = " ".repeat(spaces) + "*".repeat(stars) + " ".repeat(spaces);
        listy.push(level);
    }
    return listy;
}

File: codewars/Build Tower/Build_Tower.cpp
#include <bits/stdc++.h>

using namespace std;

void towerBuilder(unsigned nFloors) {
    vector <string> V(nFloors, "*");
    for (int i = 0; i < nFloors; i++) {
        string a, b;
        for (int j = i; j < nFloors - 1; j++) a += "-";
        for (int k = 0; k < i; k++) b += "*";
    V[i] = a + b + V[i] + b + a;
    cout << V[i] << endl;
    }
}